openwisp_controller/config/base/vpn.py

```diff
@@ -113,8 +113,8 @@ class AbstractVpn(ShareableOrgMixinUniqueName, BaseConfig):
         '-----END DH PARAMETERS-----\n'
     )
     # needed for wireguard
-    public_key = models.CharField(blank=True, max_length=44)
-    private_key = models.CharField(blank=True, max_length=44)
+    public_key = models.CharField(blank=True, null=True, max_length=44)
+    private_key = models.CharField(blank=True, null=True, max_length=44)
```

Comment:
since we already have the possibility of setting it to be blank, I would not add `null=True`, I would simply set these two fields to an empty string. Having both empty strings and null is not useful, we can simplify this part.